### PR TITLE
typo in ISSUE_TEMPLATE.md

### LINKED ISSUE #1130: Typo in ISSUE_TEMPLATE.md
# Description
Typo in ISSUE_TEMPLATE.md file

## Steps to Reproduce
spelling error in word 'below'

### LINKED ISSUE #1132: Typo in README.md file
Description
Typo in README.md file

Steps to Reproduce
Spelling and different grammatical errors in the whole document.

### PR DESCRIPTION
**Pull request by anuragsachdeva28**
# Description

Fixes #1130  
Fixes #1132 

## Status
**READY** 

## Type of change
- Documentation (fix documentation)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation (if applicable)

### CODE REVIEW

**Comment by anuragsachdeva28** [comment]
@FabioRosado, Hey, I just went by the rules of grammar. But if you say, I  will correct them and commit new changes so that other typos can be corrected.

File: README.md [hunk 1]
```diff
@@ -37,10 +37,10 @@
 
 ---
 
-An open source chat bot framework written in Python. It is designed to be extendable, scalable and simple.
+An open-source chatbot framework written in Python. It is designed to be extendable, scalable and simple.
```
Comment (jacobtomlinson) [inline README.md]: Correct adjective use of `open-source`.

File: README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ In this new frontier of DevOps, it is becoming more and more popular to interact
 
 ## Why use opsdroid?
 
- * It's open source
+ * It's open-source
```
Comment (jacobtomlinson) [inline README.md]: Incorrect noun use, should be `open source`

**Comment by anuragsachdeva28** [comment]
@jacobtomlinson  Thanks for your suggestions. I have made respective changes.

**Comment by jacobtomlinson** [comment]
Thanks for taking the time to raise this.

I'm going to merge this PR as is does improve the docs. However, this kind of low effort PR does not meet the [hacktoberfest quality standards](https://hacktoberfest.digitalocean.com/details#quality-standards) so I am marking as invalid.

**Pull request merged**